.detekt.yml

```diff
@@ -33,6 +33,9 @@ formatting:
     active: false
   FinalNewline:
     active: false
+  FunctionSignature:
```

Comment:
> Note that this commit introduces some indentation issues which will be fixed in the following commit.

To a reviewer it might be unclear why theses issues were introduced in the first place. The missing link is that the reformatting was automatically done by `ktlint`, which introduced some temporary side effects. So, maybe mention the command that was run to apply the reformatting as part of the commit message?